src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -93,20 +99,42 @@ internal abstract partial class DefinitionItem
         internal abstract bool IsExternal { get; }
 
         protected DefinitionItem(
-            ImmutableArray<string> tags,
-            ImmutableArray<TaggedText> displayParts,
-            ImmutableArray<TaggedText> nameDisplayParts,
-            ImmutableArray<TaggedText> originationParts,
-            ImmutableArray<DocumentSpan> sourceSpans,
-            ImmutableDictionary<string, string> properties,
-            bool displayIfNoReferences)
+          ImmutableArray<string> tags,
+          ImmutableArray<TaggedText> displayParts,
+          ImmutableArray<TaggedText> nameDisplayParts,
+          ImmutableArray<TaggedText> originationParts,
+          ImmutableArray<DocumentSpan> sourceSpans,
+          ImmutableDictionary<string, string> properties,
+          bool displayIfNoReferences) :
+            this(
+                tags,
+                displayParts,
+                nameDisplayParts,
+                originationParts,
+                sourceSpans,
+                properties,
+                ImmutableArray<AdditionalProperty>.Empty,
+                displayIfNoReferences)
+        {
+        }
+
+        protected DefinitionItem(
+        ImmutableArray<string> tags,
+        ImmutableArray<TaggedText> displayParts,
+        ImmutableArray<TaggedText> nameDisplayParts,
+        ImmutableArray<TaggedText> originationParts,
+        ImmutableArray<DocumentSpan> sourceSpans,
+        ImmutableDictionary<string, string> properties,
+        ImmutableArray<AdditionalProperty> displayableProperties,
+        bool displayIfNoReferences)
         {
             Tags = tags;
             DisplayParts = displayParts;
             NameDisplayParts = nameDisplayParts.IsDefaultOrEmpty ? displayParts : nameDisplayParts;
             OriginationParts = originationParts.NullToEmpty();
             SourceSpans = sourceSpans.NullToEmpty();
             Properties = properties ?? ImmutableDictionary<string, string>.Empty;
+            DisplayableProperties = displayableProperties;
```

Comment:
Seems that `displayableProperties` can be `default` for one of the callers below.

```suggestion
            DisplayableProperties = !displayableProperties.IsDefault ? displayableProperties : ImmutableArray<AdditionalProperty>.Empty;
```